custom/icds_reports/utils/vhsnd_datapull/sql_query_vhsnd.sql

```diff
@@ -0,0 +1,17 @@
+COPY(
+SELECT state_name,
+    district_name,
+    block_name,
+    supervisor_name,
+    awc_name,
+    awc_site_code,
+    vhsnd_date_past_month
+FROM agg_awc_monthly INNER JOIN "ucr_icds-cas_static-vhnd_form_28e7fd58" vhnd_ucr ON agg_awc_monthly.awc_id=vhnd_ucr.awc_id AND vhnd_ucr.vhsnd_date_past_month>='2020-02-01' AND vhnd_ucr.vhsnd_date_past_month<'2020-03-01'
```

Comment:
what is agg_awc_monthly used for? is it just the locations? if so awc_location is probably a better fit

Reply:
yeah, But we want to only include launched awcs and that info is in agg_awc table.

Reply:
oh man thats terrible, but makes sense